sdk/nodejs/provider/provider.ts

```diff
@@ -158,7 +158,7 @@ export interface Provider {
      * @param olds The old input properties to use for validation.
      * @param news The new input properties to use for validation.
      */
-    check?: (urn: resource.URN, olds: any, news: any, sequenceNumber: number) => Promise<CheckResult>;
+    check?: (urn: resource.URN, olds: any, news: any) => Promise<CheckResult>;
```

Comment:
This file is for full fledged providers, but what about the dynamic provider?

https://github.com/pulumi/pulumi/blob/43587cee92b821335a7b5fb996c12e708b477660/sdk/nodejs/dynamic/index.ts#L121